source/clear-linux/reference/system-requirements.rst

```diff
@@ -3,43 +3,76 @@
 Recommended minimum system requirements
 #######################################
 
-|CL-ATTR| can run on very minimal hardware. For example, it can run on a
-system with a single core CPU, 128MB of memory, and 600MB of disk space.
+|CL-ATTR| can run on most modern hardware and is capable of running on very
+minimal hardware resources. For example, it can even run on a system with a
+single core CPU, 128MB of memory, and 600MB of disk space.
 
 Different use cases and applications will require different configurations.
-For general |CL| use, in addition to a supported processor, the recommended
-minimum requirements include:
 
-*  Processors:
 
-   |CL-ATTR| can run on any Intel® 64bit processors which support UEFI\*
-   and SSE\* v4.1 streaming SIMD\* instructions.
+Installer requirements
+**********************
 
-   The following processor families have been verified to run |CL|:
+The *live desktop* installer requires at least 1 GB of RAM because more
+resources are required to run in live mode than after |CL| is installed onto
+persistent storage.
 
-   * 2nd Generation, or later, Intel® Core™ processor family.
-   * Intel® Xeon® Processor E3
-   * Intel® Xeon® Processor E5
-   * Intel® Xeon® Processor E7
-   * Intel® Atom™ processor C2000 product family for servers -- Q3 2013
-     version or later.
-   * Intel® Atom™ processor E3800 series -- Q4 2013 version or later.
+For hardware with less resources, use the *live server* installer because it
+has a smaller memory footprint.
 
-   To help determine if a processor is supported, you can run a
-   :ref:`compatibility check<compatibility-check>` or go to
-   http://ark.intel.com and check for these features:
+See https://clearlinux.org/downloads for more download options.
 
-   * Instruction Set = 64-bit
-   * Instruction Set Extensions = SSSE3
-   * Instruction Set Extensions = SSE 4.1
-   * Instruction Set Extensions = SSE 4.2
-   * Instruction Set Extensions = AES
-   * Instruction Set Extensions = PCLMUL
 
-*  Memory: 4GB RAM
+System requirements
+*******************
 
-*  Hard Disk: 20GB HDD
+|CL| requires an x86 64-bit processor which supports and SSE\* v4.1 streaming
+SIMD\* instructions, and a system which supports UEFI\*.
 
-*  Network: Active Internet connection
+To help determine if a processor is supported, you can run a
+:ref:`compatibility check<compatibility-check>` or go to http://ark.intel.com
+and check for these features:
+
+* Instruction Set:
+
+  - 64-bit
+  
+* Instruction Set Extensions:
+
+  - SSSE3
+  - SSE 4.1
+  - SSE 4.2
+  - AES
+  - PCLMUL
+
+The following processor families have been verified to run |CL|:
+
+* Intel® Core™ processor family (2nd Generation or later)
+* Intel® Xeon® Processor E3
```

Comment:
Please change verbiage as follows: 
From: "Processor E3" to "E3-xxxx processor"
Apply this to E5 and E7 descriptions too.

Reply:
I have tidied up this section altogether by splitting by process-type instead of individual series and representing in a table.
 
I am also trying to be cognizant of the branding in place for each at https://ark.intel.com/content/www/us/en/ark.html#@Processors 

The terminology of Generation/Family/Series is used depending what product is being talked about. Let me know if you think this can be improved further.

Reply:
I appreciate you creating a separate table for processors.  We can keep that.  Our branding database dictates usage of specific noun-phrases re processors, not me.  We cannot simply state  "E3" per this database. Instead we must only use, literally: 
- "E3-xxxx processor"
- "E5-xxxx processor"
- "E7-xxxx processor"

Please apply these changes here.

Reply:
@puneetse. Hold please. Will report back within the next hour.